src/diff_tform.c

```diff
@@ -318,8 +318,10 @@ static int normalize_find_opts(
 #undef USE_DEFAULT
 
 	if (!opts->rename_limit) {
-		opts->rename_limit = git_config__get_int_force(
-			cfg, "diff.renamelimit", DEFAULT_RENAME_LIMIT);
+		if (diff->repo) {
```

Comment:
I'd also change those above, it's too roundabout when we care about the config existing.